CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.22)
+project(
+  cps-config
+  LANGUAGES CXX
+  VERSION 0.0.1)
```

Comment:
I'd prefer not to hardcode the version separately in every supported build system, I opened #83, which adds a `VERSION` file, I think you can use `file(READ ...)` to accomplish the same basic thing in CMake?

Reply:
Makes total sense. I'll rebase and refactor once #83 is merged.

Reply:
Merged :)